### PR TITLE
Featrue: nuke container / test

### PR DESCRIPTION
**Pull request by allmywant**
1. Nuke/Restore the container
2. Start/Stop/Archive tests

### CODE REVIEW

File: bitbucket-pipelines.yml [hunk 1]
```diff
@@ -13,16 +49,16 @@ pipelines:
...
+      - step:
+          name: Invalidate Cloudfront objects
+          script:
+            - pipe: atlassian/aws-cloudfront-invalidate:0.4.1
+              variables:
+                AWS_ACCESS_KEY_ID: $AWS_ACCESS_KEY_ID
+                AWS_SECRET_ACCESS_KEY: $AWS_SECRET_ACCESS_KEY
+                AWS_DEFAULT_REGION: $AWS_DEFAULT_REGION
+                DISTRIBUTION_ID: '$AWS_CF_DISTRIBUTION'
+                PATHS: '$AWS_CF_PATHS'
```
Comment (kingo55) [inline bitbucket-pipelines.yml]: Can you please revert this? It's a configured part of the Mint pipeline, and not everyone will have CF invalidation setup.

**Review by kingo55 — COMMENTED** [review]
@allmywant - thanks for sending through this PR. Will just need to revert part of the Mint pipeline and I think we can get this merged!

File: bitbucket-pipelines.yml (hunk 1 above)
Comment (allmywant) [reply bitbucket-pipelines.yml]: @kingo55 
Reverted.

Comment (kingo55) [reply bitbucket-pipelines.yml]: Thanks @allmywant - perfect...

**Review by kingo55 — APPROVED** [review]

**Review by dapperdrop — APPROVED** [review]

**Pull request merged**